### PR TITLE
add Venhancer inference

### PR DESCRIPTION
**Pull request by Songyuanwei**
# What does this PR do?

add venhancer inference.

Fixes # (issue)

Adds # (feature)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/venhancer/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  </tr>
+  <tr>
+  <td>
+    <img src=assets/input_astronaut.gif width="380">
+  </td>
+  <td>
+    <img src=assets/output_astronaut.gif width="380">
+  </td>
+  </tr>
+
```
Comment (SamitHuang) [inline examples/venhancer/README.md:42]: wrong table format. missing ending <>

Comment (Songyuanwei) [reply examples/venhancer/README.md:32]: done

**Comment by CaitinZhao** [comment]
图片不要放代码仓

**Review by SamitHuang — COMMENTED** [review]
不放在assets，直接拖到readme编辑页

**Review by SamitHuang — APPROVED** [review]

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**